symphony-bdk-core/src/main/java/com/symphony/bdk/core/SymphonyBdk.java

```diff
@@ -42,6 +49,7 @@
 public class SymphonyBdk {
 
   private final BdkConfig config;
+  private final List<Extension> extensions;
```

Comment:
what is the value of storing over instanciating them on the fly in the getExtension() method? 
I would say to keep the same lifecycle as other services (but just to make sure and to open the discussion)

Reply:
Indeed, the added value is low and we could instantiate them on the fly.